x-pack/plugins/triggers_actions_ui/public/application/sections/alert_form/alert_notify_when.tsx

```diff
@@ -62,22 +62,22 @@ const NOTIFY_WHEN_OPTIONS: Array<EuiSuperSelectOption<AlertNotifyWhenType>> = [
     inputDisplay: i18n.translate(
       'xpack.triggersActionsUI.sections.alertForm.alertNotifyWhen.onActiveAlert.display',
       {
-        defaultMessage: 'Every time rule is active',
+        defaultMessage: 'Every time alert is active',
```

Comment:
@gmmorris I also changed this  from `rule` to `alert` because I believe that's more correct. Is that right?

Reply:
Yup, looks right to me :)